sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala

```diff
@@ -65,6 +65,11 @@ object HiveResult {
     // database, table name, isTemp.
     case command @ ExecutedCommandExec(s: ShowTablesCommand) if !s.isExtended =>
       command.executeCollect().map(_.getString(1))
+    // SHOW TABLE EXTENDED in Hive do not have isTemp while our v1 command outputs isTemp.
+    case command @ ExecutedCommandExec(s: ShowTablesCommand) if s.isExtended =>
+      command.executeCollect().map(_.getMap(3))
+        .map(m => m.keyArray().array.zip(m.valueArray().array)
+          .map(kv => s"${kv._1}: ${kv._2}").mkString("\n"))
```

Comment:
https://github.com/apache/spark/pull/32563/files#r635773394, there seems no `space` after the colon in hive results？